praise-bot/functions/save_praise.ts

```diff
@@ -0,0 +1,105 @@
+import { DefineFunction, Schema, SlackFunction } from "deno-slack-sdk/mod.ts";
+import { SlackAPIClient } from "deno-slack-sdk/types.ts";
+import { isDebugMode } from "./internals/debug_mode.ts";
+
+/**
+ * Defines the SavePraiseFunction input and output parameters
+ */
+export const SavePraiseFunction = DefineFunction({
+    callback_id: "save_praise",
+    title: "Save a Praise",
+    description: "Saves data about a praise based on user input choices",
+    source_file: "functions/save_praise.ts", // The file with the exported function handler
+    input_parameters: {
+      properties: {
+        sender: {
```

Comment:
@AndrewHUNGNguyen looks like this then is the variable you want

Reply:
Yes the sender variable is what I want to send as data about the User that triggered the praise slackbot